### ISSUE TITLE
Add stats per ball type

### ISSUE THREAD

**Issue by jarekb84** [PoochyEXE/PoochyEXE.github.io#17]
Would be great to see points earned by each ball type, maybe to see which are more worthwhile to upgrade. 

Can extend this further by adding stats per time fame

```
                          Points earned in last
           Balls Dropped  5 sec           15 sec          60 sec          10 min           30 min             1 hour             All Time            
Total      102,997        10,299,597,003  30,898,791,009  92,696,373,027  926,963,730,270  2,780,891,190,810  5,561,782,381,620  16,685,347,144,860  
Gold       17,321         8,660,500       25,981,500      77,944,500      779,445,000      2,338,335,000      4,676,670,000      14,030,010,000      
Ruby       4,710          23,550,000      70,650,000      211,950,000     2,119,500,000    6,358,500,000      12,717,000,000     38,151,000,000      
Sapphire   4,079          203,950,000     611,850,000     1,835,550,000   18,355,500,000   55,066,500,000     110,133,000,000    330,399,000,000     
Emerald    5,252          262,600,000     787,800,000     2,363,400,000   23,634,000,000   70,902,000,000     141,804,000,000    425,412,000,000     
Topaz      4,147          2,073,500,000   6,220,500,000   18,661,500,000  186,615,000,000  559,845,000,000    1,119,690,000,000  3,359,070,000,000   
Turquoise  3,716          74,320,000      222,960,000     668,880,000     6,688,800,000    20,066,400,000     40,132,800,000     120,398,400,000     
Amethyst   3,452          103,560,000     310,680,000     932,040,000     9,320,400,000    27,961,200,000     55,922,400,000     167,767,200,000     
Opal       2,359          94,360,000      283,080,000     849,240,000     8,492,400,000    25,477,200,000     50,954,400,000     152,863,200,000     
8          1,386          110,880,000     332,640,000     997,920,000     9,979,200,000    29,937,600,000     59,875,200,000     179,625,600,000     
Beach      616            55,440,000      166,320,000     498,960,000     4,989,600,000    14,968,800,000     29,937,600,000     89,812,800,000      

```

**Comment by PoochyEXE** [comment]
I like the idea, and I've been meaning to expand the stats panel, but I'll need to figure out how to do this in a way where the UI still looks OK and not too cluttered. I'll plan on coming back to this one later.

**Comment by PoochyEXE** [comment]
Partly implemented in v1.4.7-beta. I added all-time totals by ball type, but I still need to figure out how to break down last 5/15/60 seconds by ball type in a way that doesn't make the stats panel look like airplane cockpit. I'm currently considering making a drop-down where you can select between all time, last 5 seconds, last 15 seconds, and last 60 seconds.

**Comment by PoochyEXE** [comment]
Finally added in v1.7.5-beta.
